packages/postgress/src/MessagePickupRepositoryPg.ts

```diff
@@ -0,0 +1,616 @@
+    this.logger?.debug(`[getLiveSessionFromDB] initializing find registry for connectionId ${connectionId}`)
+    if (!connectionId) throw new Error('connectionId is not defined')
+    try {
+      const queryLiveSession = await this.messagesCollection?.query(
+        `SELECT * FROM storelivesession WHERE connectionid = $1 LIMIT $2`,
+        [connectionId, 1],
+      )
+      // Check if liveSession is not empty (record found)
+      const recordFound = queryLiveSession && queryLiveSession.rows && queryLiveSession.rows.length > 0
+      this.logger?.debug(`[getLiveSessionFromDB] record found status ${recordFound} to connectionId ${connectionId}`)
+      return recordFound ? queryLiveSession.rows[0] : false
+    } catch (error) {
+      this.logger?.debug(`[getLiveSessionFromDB] Error find to connectionId ${connectionId}`)
+      return false // Return false in case of an error
+    }
+  }
+
+  /**
+   * This method adds a new connectionId and instance name to DB upon LiveSessionSave event
+   * @param connectionId
+   * @param instance
+   */
+  private async addLiveSessionFromDb(session: MessagePickupSession, instance: string): Promise<void> {
+    const { id, connectionId } = session
+    this.logger?.debug(`[addLiveSessionFromDb] initializing add LiveSession DB to connectionId ${connectionId}`)
+    if (!session) throw new Error('session is not defined')
+    try {
+      const insertMessageDB = await this.messagesCollection?.query(
+        'INSERT INTO storelivesession (sessionid, connectionid, instance) VALUES($1, $2, $3) RETURNING sessionid',
+        [id, connectionId, instance],
+      )
+      const liveSessionId = insertMessageDB?.rows[0].sessionid
+      this.logger?.debug(`[addLiveSessionFromDb] add liveSession to ${connectionId} and result ${liveSessionId}`)
+    } catch (error) {
+      this.logger?.debug(`[addLiveSessionFromDb] error add liveSession DB ${connectionId}`)
+    }
+  }
+
+  /**
+   *This method remove connectionId record to DB upon LiveSessionRemove event
+   * @param connectionId
+   */
+
+  private async removeLiveSessionFromDb(connectionId: string): Promise<void> {
+    this.logger?.debug(`[removeLiveSessionFromDb] initializing remove LiveSession to connectionId ${connectionId}`)
+    if (!connectionId) throw new Error('connectionId is not defined')
+    try {
+      // Construct the SQL query with the placeholders
+      const query = `DELETE FROM storelivesession WHERE connectionid = $1`
+
+      // Add connectionId  for query parameters
+      const queryParams = [connectionId]
+
+      await this.messagesCollection?.query(query, queryParams)
+
+      this.logger?.debug(`[removeLiveSessionFromDb] removed LiveSession to connectionId ${connectionId}`)
+    } catch (error) {
+      this.logger?.error(`[removeLiveSessionFromDb] Error removing LiveSession: ${error}`)
+    }
+  }
+
+  private async pushNotification(token: string, messageId: string): Promise<void> {
```

Comment:
This can be removed and left to be implemented by the controller of this object, to use whatever push notification method they want.